sys/include/bcd.h

```diff
@@ -64,11 +66,48 @@ static inline uint8_t bcd_to_byte(uint8_t bcd)
  * @param[in]  len  Size of the destination buffer
  *
  * @return number of bytes written
- * @return -ENOBUFS if @p dst is not large enough
+ * @retval -ENOBUFS if @p dst is not large enough
  *         In that case the state of @p dst is undefined.
  */
 int bcd_buf_from_u32(uint32_t val, void *dst, size_t len);
 
+/**
+ * @brief   Convert a BCD buffer into it's binary representation
+ *          (This will reverse @see bcd_buf_from_u32)
```

Comment:
```suggestion
 *          (This will reverse @ref bcd_buf_from_u32)
```
![image](https://github.com/user-attachments/assets/882f1993-024a-4058-9678-72f60ec61fa5)